.github/workflows/release.yml

```diff
@@ -80,3 +80,24 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.ASGARDIO_GITHUB_BOT_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      
+      - name: Create samples zip file
+        run: |
+          mkdir -p /tmp/samples
+          
+          cp -r samples/asgardeo-react-app /tmp/samples/
+          cp -r samples/asgardeo-choreo-react-express /tmp/samples/
+
+          cd /tmp
+          zip -r asgardeo-samples.zip samples \
```

Comment:
Instead of bundling all samples into a single zip file, should we publish each sample as an individual zip file? Ideally, the release artifacts should follow the same structure as in https://github.com/asgardeo/asgardeo-auth-react-sdk/releases/tag/v5.2.3